crates/autopilot/src/database.rs

```diff
@@ -33,7 +35,7 @@ struct Metrics {
 
     /// Timing of db queries.
     #[metric(labels("type"))]
-    database_queries: prometheus::HistogramVec,
+    autopilot_database_queries: prometheus::HistogramVec,
```

Comment:
This metric would have the same name as the one in the orderbook crate which would then error when registering with the global prometheus registry during e2e tests.